modules/ROOT/pages/backup-and-disaster-recovery.adoc

```diff
@@ -27,6 +27,22 @@ Specifies the deadline for the backup job, for example, 30s or 5m. Default is 20
 +
 Outputs logs to a file.
 
+Backup contents:
+
+[%header%autowidth.spread]
+|===
+| Information | Included in PCE Backup 
+| Databases | Y
+| Configurations | Y
+| Secrets | Y
+| Object Store | Y
+| Drives | N
+| NFS Server Configuration | N
```

Comment:
It's not that server configuration that is not included, but the NFS files

Reply:
Got it - thank you!